### PR TITLE
 SOFTWARE-3276 and SOFTWARE-3281 remove GOC references, use osg-notify

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -36,12 +36,12 @@ pages:
     - How to Cut a Release: 'release/cut-sw-release.md'
     - How to Cut a Data Release: 'release/cut-data-release.md'
     - Release Policy: 'release/release-policy.md'
-    - Release Schedule: 'release/release-schedule.md'
     - New Release Series: 'release/new-release-series.md'
     - Old Release Series Removal: 'release/old-release-removal.md'
     - ITB Prerelease Testing: 'release/itb-testing.md'
     - Empty Packages: 'release/empty-pkgs.md'
     - Acceptance Testing: 'release/acceptance-testing.md'
+    - Sending Announcements: 'release/sending-announcements.md'
```
Comment (brianhlin) [inline mkdocs.yml]: Is there a sending-announcements page that you need to add to the PR?

File: docs/release/cut-sw-release.md [hunk 2]
```diff
@@ -318,9 +266,17 @@ The following instructions are meant for the release manager (or interim release
 
         We welcome feedback on this release!
 
-2.  The release manager emails the announcement to `vdt-discuss@opensciencegrid.org`
-3.  The release manager asks the GOC to distribute the announcement by [opening a ticket](https://ticket.opensciencegrid.org/goc/other)
-4.  The release manager closes the tickets marked 'Ready for Release' in the release's JIRA filter using the 'bulk change' function.
+2.  The release manager uses the [osg-notify tool](https://opensciencegrid.org/operations/services/sending-announcements/) to send the release announcement using the following command:
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: As discussed, let's change this line to just say that the following command needs to be run from `submit-1`? Also, do our external S&R members have access to this host?

Comment (timtheisen) [reply docs/release/cut-sw-release.md]: The release manager or his designated rep does this. I think it is not difficult to get an account on submit-1.chtc.wisc.edu.

Comment (timtheisen) [reply docs/release/cut-sw-release.md]: submit-1 has the prerequisites. You still have to setup the git repository as described in the linked document.

Comment (timtheisen) [reply docs/release/cut-sw-release.md]: submit-1 has the *software* prerequisites, it is *not* in the DNS SPF record.

Comment (timtheisen) [reply docs/release/cut-sw-release.md]: @djw8605 : When will the SPF record be updated?

File: mkdocs.yml (hunk 1 above)
Comment (timtheisen) [reply mkdocs.yml]: Old cruft. Deleted.

File: docs/release/cut-sw-release.md (hunk 2 above)
Comment (djw8605) [reply docs/release/cut-sw-release.md]: Hi, sorry I didn't get this notification.  Anyways, do you have an ip range for the DNS SPF record.  I can add it to the DNS entry then.

Comment (brianhlin) [reply docs/release/cut-sw-release.md]: @timtheisen please update the line now that the SPF record has been updated

**Pull request merged**